cmd/oras/root/push.go

```diff
@@ -183,6 +194,22 @@ func runPush(cmd *cobra.Command, opts *pushOptions) error {
 		}
 		desc.Annotations = packOpts.ConfigAnnotations
 		packOpts.ConfigDescriptor = &desc
+	} else if opts.Platform.Platform != nil {
+		blob, err := json.Marshal(opts.Platform.Platform)
+		if err != nil {
+			return err
+		}
+		desc := ocispec.Descriptor{
+			MediaType: oras.MediaTypeUnknownConfig,
+			Digest:    digest.FromBytes(blob),
+			Size:      int64(len(blob)),
+		}
```

Comment:
nit: This part can be replaced by `func NewDescriptorFromBytes(mediaType string, content []byte) ocispec.Descriptor` of oras-go. Reference: [NewDescriptorFromBytes](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/content#NewDescriptorFromBytes)

Reply:
Done